stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -34,9 +34,33 @@ spec:
       containers:
       - name: fluent-bit
         image: "{{ .Values.image.fluent_bit.repository }}:{{ .Values.image.fluent_bit.tag }}"
+{{- if .Values.input.kmsg.enabled }}
+        securityContext:
+          privileged: true
+{{- end }}
         imagePullPolicy: "{{ .Values.image.pullPolicy }}"
         env:
-{{ toYaml .Values.env | indent 10 }}
```

Comment:
You broke the ability for anyone to set env vars without this. Please add it back.

Reply:
It is back.